BPMobileMessaging/Classes/ConactCenterCommunication/ContactCenterCommunicating.swift

```diff
@@ -4,74 +4,79 @@
+    /// Returns all client events and all server events for all sessions related to the CRM case defined by the scenario which handles a current chat session. For each session, multiple event objects can be returned; each event's timestamp attribute can be used to restore the correct message order. Can be called after receiving `chatSessionCaseSet`; will return `chatSessionCaseNotSpecified` if server scenario did not specify the case. Note that the case could be specified later during the scenario execution; not necessarily immediately after session start.
     /// - Parameters:
     ///   - chatID: The current chat ID
     ///   - completion: Chat sessions with client and server events [ContactCenterChatSession](x-source-tag://ContactCenterEvent) or [ContactCenterError](x-source-tag://ContactCenterError) otherwise
     /// - Tag: getCaseHistory
     func getCaseHistory(chatID: String, with completion: @escaping ((Result<[ContactCenterChatSession], Error>) -> Void))
-    /// Request Chat initiates a chat session. It provides values of all or some of the expected parameters, and it may also contain the phone number of the mobile device. Note that if the mobile scenario entry is not configured for automatic callback, the agent can still use this number to call the mobile user manually, either upon the agent's own initiative or when asked to do this via a chat message from the mobile user.
-    /// - Parameters:
-    ///   - phoneNumber: phone number for callback, if necessary
-    ///   - from: Propagated into scenario variable $(item.from). May be used to specify either the device owner’s name or phone number.
-    ///   - parameters: Additional parameters.
-    ///   - completion: Returns chat session properties that includes `chatID` in [ContactCenterChatSessionProperties](x-source-tag://ContactCenterChatSessionProperties) or [ContactCenterError](x-source-tag://ContactCenterError) otherwise
-    /// - Tag: requestChat
-    func requestChat(phoneNumber: String, from: String, parameters: [String: String], with completion: @escaping ((Result<ContactCenterChatSessionProperties, Error>) -> Void))
-    /// Send a chat message. Before message is sent the function generates a `messageID` which is returned in a completion
+    /// Send a chat message. Before message is sent the function generates a `messageID` which is returned in a completion. This `messageID` should be later used to match the `chatSessionMessageDelivered` and `chatSessionMessageRead` server events which notify the application that the message has been delivered to or read by an agent.
     /// - Parameters:
     ///   - chatID: The current chat ID
     ///   - message: Text of the message
     ///   - completion: Returns  `messageID` in the format chatId:messageNumber where messageNumber is ordinal number of the given message in the chat exchange or [ContactCenterError](x-source-tag://ContactCenterError) otherwise
```

Comment:
```suggestion
    ///   - completion: Returns  `messageID` in the format chatId:messageNumber where messageNumber is
    /// ordinal number of the given message in the chat exchange or [ContactCenterError](x-source-tag://ContactCenterError) otherwise
```